esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -1674,6 +1674,45 @@ void CTpWrapper::getDropZoneMachineList(double clientVersion, bool ECLWatchVisib
 
 void CTpWrapper::getTpDropZones(double clientVersion, const char* name, bool ECLWatchVisibleOnly, IArrayOf<IConstTpDropZone>& list)
 {
+
+#ifdef _CONTAINERIZED
+    IPropertyTree * storage = queryGlobalConfig().queryPropTree("storage");
+    if (storage)
+    {
+        StringBuffer pname("planes[@isDropZone='true']");
+        if (name && *name)
+            pname.appendf("[@name='%s']", name);
+        Owned<IPropertyTreeIterator> planes = storage->getElements(pname.str());
```

Comment:
Feels like getDropZones(const char *optName) should be a utility function for general use, outside of esp code, that esp code calls. And needed by dfurun for example.

Utility method could return a list of matching plane IPT's.